doc/man1/openssl-verify.pod.in

```diff
@@ -9,12 +9,12 @@ openssl-verify - certificate verification command
 
 B<openssl> B<verify>
 [B<-help>]
-[B<-CRLfile> I<file>]
+[B<-CRLfile> I<filename>|I<uri>]
 [B<-crl_download>]
 [B<-show_chain>]
 [B<-verbose>]
-[B<-trusted> I<file>]
-[B<-untrusted> I<file>]
+[B<-trusted> I<filename>|I<uri>]
+[B<-untrusted> I<filename>|I<uri>]
```

Comment:
Do these really support URIs now?

Reply:
Well, this works to some extent, at least for `file:` with absolute paths:
```
apps/openssl verify \
  -trusted file:/home/david/openssl/prepare/test/certs/root-cert.pem \
  -untrusted file:/home/david/openssl/prepare/test/certs/ca-cert.pem \
  file:/home/david/openssl/prepare/test/certs/ee-pss-sha256-cert.pem 
file:/home/david/openssl/prepare/test/certs/ee-pss-sha256-cert.pem: OK
```

@levitte, would it be hard to support the `http://` scheme via OSSL_STORE as well?

Reply:
It supports uris that OSSL_STORE supports, doesn't it? Maybe mention that somewhere?

Reply:
It looks like this has not yet been done for any app where this generalization has been reflected in its documentation in the same way as here for `verify`.
Likely this would best be done similarly to the meanwhile generic doc of input format options in `doc/man1/openssl-format-options.pod`.

Reply:
I've meanwhile filed an issue for there points: #14001.